File: JH/10950.py
import sys

n = int(input())
for i in range(n):
  a, b = map(int, sys.stdin.readline().split())
  print(a+b)
  
# 여러 줄 입력 받는 경우 input() -> 시간초과 
# sys.stdin.readline(): 한 줄 단위로 입력 (\n포함)
# 문자열 형태로 저장

File: Lami/01.17/10950.js
const fs = require('fs');
const filePath = process.platform === 'linux' ? '/dev/stdin' : './input.txt';
let input = fs.readFileSync(filePath).toString().split('\n');

function solution() {
  for (let i = 1; i <= input[0]; i++) {
    let numbers = input[i].split(' ');

    console.log(Number(numbers[0]) + Number(numbers[1]));
  }
}

solution();
